client/src/components/shared/header.tsx

```diff
@@ -0,0 +1,37 @@
+import AppBar from "@mui/material/AppBar";
+import Box from "@mui/material/Box";
+import Toolbar from "@mui/material/Toolbar";
+import Typography from "@mui/material/Typography";
+import Button from "@mui/material/Button";
+
+const Header = () => {
+  return (
+    <Box flexGrow={1}>
+      <AppBar position="static" >
+        <Toolbar sx={{ marginX: "50px" }}>
+
+          <Box flexGrow={1} display={"inline-flex"}>
+            <img src="/logo.svg" alt="logo" />
+            <Typography color={"primary"} marginX={1} variant="h6" textTransform={"uppercase"}>
+              Colosseum
+            </Typography>
+          </Box>
+
+          <Box>
```

Comment:
Let's match the design proposal, there the `Tabs` are used

Reply:
@MarioStojcevski please have a look at https://github.com/42dotmk/colosseum/pull/8 before doing this change, I added these changes there to enable the routing